spec/fabricators/user_fabricator.rb

```diff
@@ -0,0 +1,7 @@
+# frozen_string_literal: true
+
+Fabricator(:user) do
+  email { FFaker::Internet.email }
+  password { 'password123' }
+  password_confirmation { 'password123' }
```

Comment:
```suggestion
  password 'password123'
  password_confirmation 'password123'
```
Should it be juts this because it's static value?

Reply:
Yup changed it 😄 Updated in https://github.com/aetheryna/google-scraper-ruby/pull/34/commits/25b9f583ac59d64e0e10127480a5f768837eeeaa